src/EditorFeatures/Test/Diagnostics/AbstractDiagnosticProviderBasedUserDiagnosticTest.cs

```diff
@@ -111,5 +114,14 @@ internal override IEnumerable<Diagnostic> GetDiagnostics(TestWorkspace workspace
                 }
             }
         }
+
+        private void AssertNoAnalyzerExceptionDiagnostics(IEnumerable<Diagnostic> diagnostics)
+        {
+            // The internal method `Microsoft.CodeAnalysis.Diagnostics.AnalyzerExecutor::IsAnalyzerExceptionDiagnostic()`
+            // does essentially this, but due to linked files between projects, this project cannot have internals visible
```

Comment:
Can you at least use crefs then so when the one method gets deleted the other works? I think this case will fall under the magic of crefs allowing internal references.